Dockerfile

```diff
@@ -1,24 +1,30 @@
-FROM ubuntu:14.04
+FROM ubuntu:20.04
 MAINTAINER Shane Canon <scanon@lbl.gov>
 
 # Update apt and install jdk and docker engine to get docker clients
+# Docker installation instructions from https://docs.docker.com/engine/install/ubuntu/
 RUN apt-get -y update && \
-    apt-get -y install openjdk-7-jdk make git ant && \
-    apt-get -y install apt-transport-https ca-certificates && \
-    apt-key adv \
-               --keyserver hkp://ha.pool.sks-keyservers.net:80 \
-               --recv-keys 58118E89F3A912897C070ADBF76221572C52609D && \
-    echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" > /etc/apt/sources.list.d/docker.list && \
-    apt-get -y update && apt-get -y install docker-engine=1.11.2-0~trusty 
+    DEBIAN_FRONTEND=noninteractive apt-get -y install tzdata && \
```

Comment:
`tzdata` asks you to set the time zone (which stops the build process) if not installed with the `non-interactive` option.